cf-java-logging-support-log4j2/src/main/java/com/sap/hcp/cf/log4j2/converter/api/Log4jContextFieldSupplier.java

```diff
@@ -6,4 +6,8 @@
 @FunctionalInterface
 public interface Log4jContextFieldSupplier extends EventContextFieldSupplier<LogEvent> {
 
```

Comment:
Maybe some explanation somewhere why exactly these values are used here.
this might be hard to find later, if there are issues with orderings